pkg/config/generate.go

```diff
@@ -328,8 +328,13 @@ func Generate(cs *acsapi.OpenShiftManagedCluster) (err error) {
 	}
 
 	if len(c.HtPasswd) == 0 {
-		if c.HtPasswd, err = makeHtPasswd("demo", "demo"); err != nil {
-			return
+		pass, err := randomString(10)
```

Comment:
if len HtPasswdPassword == 0, generate the name
then if the given password doesn't correspond with the .htpasswd file (check bcrypt.CompareHashAndPassword), regenerate htpasswd file.

Reply:
@jim-minter please elaborate on this. Not very clear what you had in mind

Reply:
something like
```
	if len(c.HtPasswdPassword) == 0 {
		if c.HtPasswdPassword, err = randomString(10); err != nil {
			return err
		}
	}

	if bcrypt.CompareHashAndPassword(c.HtPasswd, []byte(c.HtPasswdPassword)) != nil {
		if c.HtPasswd, err = makeHtPasswd("osadmin", c.HtPasswdPassword); err != nil {
			return err
		}
	}
```